common/Jackdmp.cpp

```diff
@@ -497,10 +496,11 @@ int main(int argc, char** argv)
 
             default:
                 fprintf(stderr, "unknown option character %c\n", optopt);
-                /*fallthru*/
+                goto destroy_server;
```

Comment:
this is changing previous behaviour.
please add printing of usage. so it works just like before.

Reply:
Fixed. Since there is no more fall-through I also moved `default:` to the bottom.